### PR TITLE
also mkdir the CACHE directory

### PR DESCRIPTION
**Pull request by groovecoder**

### CODE REVIEW

**Review by Micheletto — APPROVED** [review]
r+

File: Dockerfile [hunk 1]
```diff
@@ -12,12 +12,13 @@ EXPOSE 8000
 
 USER app
 
+COPY ./requirements.txt /app/requirements.txt
+RUN pip install -r requirements.txt
+
```
Comment (groovecoder) [inline Dockerfile:17]: This helps `docker build` cache the results of the install to speed up builds.

File: Dockerfile [hunk 2]
```diff
@@ -12,12 +12,13 @@ EXPOSE 8000
...
 COPY . /app
 COPY .env-dist /app/.env
 
-RUN pip install -r requirements.txt
-
-RUN mkdir -p /app/static
+RUN mkdir -p /app/static/CACHE
```
Comment (groovecoder) [inline Dockerfile:21]: Hmm ... we may not need this at all, if we `COPY . /app`, then `static/CACHE` should now copy over.

**Review by Micheletto — APPROVED** [review]
r+

**Pull request merged**